setup.cfg

```
@@ -33,7 +32,7 @@ install_requires =
     attrs>=19,<22
     mdurl~=0.1
     typing_extensions>=3.7.4;python_version<'3.8'
-python_requires = ~=3.6
+python_requires = ~=3.7
```

```suggestion
python_requires = >=3.7
```

I'd recommend this change. If Python 4.0 ever arrives it [won't be](https://twitter.com/gvanrossum/status/1306082472443084801) anything like Python 2 -> Python 3 transition. Also, Python "minor" versions contain breaking changes so there's really no promise that e.g. Python 3.12 will be compatible either.

---

Randomly commenting on this PR... The above change would be great because the upper bound makes it difficult to add `markdown-it-py` as a dependency to other packages that do not have the same constraint. It's my understanding that the practice of setting upper bounds is generally dissuaded unless there's a strong reason for it (e.g., [PyPA PR](https://github.com/pypa/packaging.python.org/pull/850), Henry Schreiner [excellent blog post](https://iscinumpy.dev/post/bound-version-constraints/))

---

Committing this as per https://github.com/executablebooks/mdit-py-plugins/issues/36#issuecomment-1033093977